pkg/operator/ceph/config/conditions.go

```diff
@@ -48,6 +48,12 @@ func ConditionExport(context *clusterd.Context, namespaceName types.NamespacedNa
 // setCondition updates the conditions of the cluster custom resource
 func setCondition(c *clusterd.Context, namespaceName types.NamespacedName, newCondition cephv1.Condition) {
 	ctx := context.TODO()
+
+	// workaround unit test failure
```

Comment:
What's the error exactly?

Reply:
test `TestAddRemoveNode` is failing because RookClientset is empty,  I think I should use fake RookClientset...

Reply:
Yes try that